decidim-budgets/app/views/decidim/budgets/projects/show.html.erb

```diff
@@ -27,6 +27,8 @@
         </div>
       </div>
     </div>
+    <%= render partial: "decidim/shared/reference", locals: { subject: project } %>
```

Comment:
I guess it's a better developer experience, you don't need to know which partial to render, you just want to print the reference (similar to `comment_for`, `attachments_for`, etc)